frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/review-and-create-step/review-and-create-step.tsx

```diff
@@ -1,224 +1,128 @@
-            <Text key={node.name} component={TextVariants.small}>
-              {node.name}
-            </Text>
-          ))}
-        </TextContent>
-      </CardBody>
-    </Card>
-  );
-
-type NodeCardProps = { nodes: WizardNodeState[] };
-
-const REVIEW_ICON_MAP = {
-  [AlertVariant.success]: GreenCheckCircleIcon,
-  [AlertVariant.warning]: YellowExclamationTriangleIcon,
-  [AlertVariant.info]: BlueInfoCircleIcon,
-  [AlertVariant.danger]: RedExclamationCircleIcon,
-};
-
-export const ReviewListTitle: React.FC<ReviewListTitleProps> = ({ text }) => (
-  <dt>
+export const ReviewItem = ({ children, title }) => (
+  <div className="odf-review-and-create__review-item">
     <TextContent>
-      <Text component={TextVariants.h4}>{text}</Text>
+      <Text component={TextVariants.h4}>{title}</Text>
     </TextContent>
-  </dt>
+    <List isPlain>{children}</List>
+  </div>
 );
 
-type ReviewListTitleProps = { text: string };
-
-export const ReviewListBody: React.FC<ReviewListBodyProps> = ({
-  children,
-  validation,
-  hideIcon = false,
-  noValue = undefined,
-}) => {
-  const { t } = useTranslation();
-
-  const alert = VALIDATIONS(validation, t);
-  const Icon = noValue
-    ? REVIEW_ICON_MAP[AlertVariant.danger]
-    : REVIEW_ICON_MAP[alert?.variant || AlertVariant.success];
-
-  return (
-    <dd className="odf-review-and-create__body">
-      {alert?.variant || !hideIcon ? (
-        <Split>
-          <SplitItem>
-            <Icon className="odf-review-and-create__icon" />
-          </SplitItem>
-          <SplitItem isFilled>
-            {children}
-            {alert?.variant ? <ValidationMessage validation={validation} /> : null}
-          </SplitItem>
-        </Split>
-      ) : (
-        children
-      )}
-    </dd>
-  );
-};
-
-type ReviewListBodyProps = {
-  children: React.ReactNode;
-  hideIcon?: boolean;
-  noValue?: boolean;
-  validation?: ValidationType;
-};
-
 export const ReviewAndCreate: React.FC<ReviewAndCreateProps> = ({ state }) => {
   const { t } = useTranslation();
   const isMultusSupported = useFlag(GUARDED_FEATURES.OCS_MULTUS);
 
-  const { storageClass, capacityAndNodes, securityAndNetwork } = state;
-  const { nodes, enableArbiter, capacity } = capacityAndNodes;
-  const { encryption, publicNetwork, clusterNetwork, kms, networkType } = securityAndNetwork;
+  const {
+    storageClass,
+    capacityAndNodes,
+    securityAndNetwork,
+    createLocalVolumeSet,
+    backingStorage,
+  } = state;
+  const { nodes, capacity } = capacityAndNodes;
+  const { encryption, kms, networkType } = securityAndNetwork;
+  const { deployment, externalStorage, type } = backingStorage;
 
-  const validations = [...capacityAndNodesValidate(nodes, enableArbiter)];
   const isNoProvisioner = storageClass.provisioner === NO_PROVISIONER;
+  const formattedCapacity = !isNoProvisioner ? `${OSD_CAPACITY_SIZES[capacity]} TiB` : capacity;
+
+  const hasEncryption = encryption.clusterWide || encryption.storageClass;
```

Comment:
this is fine, one more alternative is   encryption.hasHandled

Reply:
I checked its for validations and by default always "true". Here we need to know if an encryption of any level is enabled or not.

Reply:
acl